src/pretix/control/templates/pretixcontrol/item/index.html

```diff
@@ -1,146 +1,169 @@
-{% extends "pretixcontrol/item/base.html" %}
-{% load i18n %}
-{% load bootstrap3 %}
-{% load formset_tags %}
-{% block inside %}
-    <form action="" method="post" class="form-horizontal" enctype="multipart/form-data">
-        {% csrf_token %}
-        <div class="row">
-            <div class="col-xs-12 col-lg-10">
-                <div class="tabbed-form">
-                    <fieldset>
-                        <legend>{% trans "General" %}</legend>
-                        {% bootstrap_field form.active layout="control" %}
-                        {% bootstrap_field form.name layout="control" %}
-                        <div class="internal-name-wrapper">
-                            {% bootstrap_field form.internal_name layout="control" %}
-                        </div>
-                        {% bootstrap_field form.category layout="control" %}
+{% extends "pretixcontrol/item/base.html" %} {% load i18n %} {% load bootstrap3
+%} {% load formset_tags %} {% block inside %}
+<form
+  action=""
+  method="post"
+  class="form-horizontal"
+  enctype="multipart/form-data"
+>
```

Comment:
Can you please revert these changes, they are unrelated. You probably have your linter/ruff set to 80 char line length